packages/components/src/components/AutoScalingInput/AutoScalingInput.tsx

```diff
@@ -71,21 +69,14 @@ export const AutoScalingInput = forwardRef<HTMLInputElement, Props>(
             if (placeholderMeasureRef?.current?.style) {
                 setPlaceholderWidth(placeholderMeasureRef.current.scrollWidth ?? 0);
             }
-        }, [setPlaceholderWidth]);
+        }, [setPlaceholderWidth, value, updateFlag]);
 
         useEffect(() => {
-            const isValueEmpty = value === '' || value === null || value === undefined;
-
-            const width = isValueEmpty
-                ? calculatedMin
-                : Math.max(calculatedMin, inputRef?.current?.scrollWidth ?? calculatedMin);
```

Comment:
This was here to cover the "empty" input (so it wont colapse to zero width). Is it working?